src/openrct2/peep/Guest.cpp

```diff
@@ -5482,7 +5482,7 @@ void Guest::UpdateWalking()
     {
         sprite = get_sprite(sprite_id);
 
-        if (sprite->generic.linked_list_index != SPRITE_LIST_PEEP)
+        if (sprite->generic.sprite_identifier != SPRITE_IDENTIFIER_PEEP)
```

Comment:
It would be neater if we used the `IsPeep()` function here.

Reply:
I'm about to replace all of this code but yes I can change that.